doc/nrf/config_and_build/dfu/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _app_dfu:
+.. _ug_fw_update:
+
+Device Firmware Updates
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Device Firmware Update (DFU) is the procedure of upgrading the application firmware version on a device.
+It is composed of two steps.
+First a new firmware image is transferred to the chip, and then the :ref:`bootloader <app_bootloaders>` tests and boots the new firmware.
+
+You can transfer the updated images to the device in two ways:
+
+* Wired, where updates are sent through a wired connection, like UART, or delivered by connecting a flash device.
+* Over-the-air (OTA), where updates are sent through a wireless connection, like Bluetooth® Low Energy.
+
+The testing and booting process depends on the choice of bootloader.
+Generally, bootloaders support two types of updates:
+
+* Direct updates, with an in-place substitution of the image.
+  In this case, it is the bootloader that must transfer the update image.
+* Background updates, where the updated image is obtained and stored by the application, but the update is completed by the bootloader after the device reboots.
+
+Based on these criteria, the |NCS| offers support for the following DFU alternatives:
+
+* For sending updates to the receiving device:
+
+  .. list-table:: DFU methods for sending updates
+    :widths: auto
+    :header-rows: 1
+
+    * - Method
+      - Description
+      - Supported transfer
+    * - :ref:`nRF Util <toolchain_management_tools>`
+      - Multi-purpose command line tool that can be used to send SMP-formatted updates to MCUmgr, among other things.
+      - Wired (SMP over UART or USB)
+    * - Cloud interfaces
+      - Each cloud comes with its specific interface to send updates to devices from the cloud. See documentation for :ref:`lib_nrf_cloud`, :ref:`lib_aws_fota` or :ref:`lib_azure_fota`.
+      - OTA (LTE, Wi-Fi)
+    * - :ref:`zephyr:mcumgr_cli`
+      - Command line tool specifically for sending updates to the MCUmgr library.
+      - Wired (SMP over UART or USB) and OTA (SMP over Bluetooth LE)
+    * - `nRF Connect Device Manager`_
+      - Mobile application for sending SMP updates over Bluetooth LE; can also be used for other SMP functionality.
+      - OTA (SMP over Bluetooth LE)
+    * - `nRF Connect for Mobile`_
+      - General purpose mobile application that can be used for sending SMP updates over Bluetooth LE, among other things.
+      - OTA (SMP over Bluetooth LE)
+    * - :ref:`MCUmgr SMP client <zephyr:mcumgr_smp_transport_specification>`
+      - SMP Client running on a microcontroller. For example, :ref:`bluetooth_central_dfu_smp`.
```

Comment:
Does the sample actually support DFU? Unless I missed something, the library allows for SMP communication and the DFU parts needs to be implemented by the integrator here.

Reply:
@hellesvik-nordic , could you have a look?

Reply:
It does not support DFU.
However, it is the simplest one I can think of.
I know however, that it is possible to add DFU to the sample, as a colleague of mine have[ done so](https://github.com/hellesvik-nordic/samples_for_nrf_connect_sdk/tree/main/bootloader_samples/client_smp/smp_client_ble).
I also know that for serial SMP Client, we support that as part of our [LWM2M client sample](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/cellular/lwm2m_client/fota_external_mcu.html).
So as far as I know we have the choice between:
SMP Client which is so simple that it does not support DFU.
SMP Client which is complicated, as it is part of a library in a LWM2M sample. But it supports DFU.

Reply:
No change made.

Reply:
Maybe we can explicitly denote that the linked sample does not support DFU, but can be used as a good implementation starting point? Otherwise it may be slightly confusing

Reply:
Edited.